components/fonts/font.rs

```diff
@@ -237,6 +235,14 @@ pub struct Font {
     /// essentially equivalent to whether or not we use it for emoji presentation.
     /// This is cached, because getting table data is expensive.
     has_color_bitmap_or_colr_table: OnceLock<bool>,
+
+    /// Whether or not this font can do fast shaping, ie whether or not it has
+    /// a kern table, but no GSUB and GPOS tables. When this is true, Servo will
+    /// shape this text without using Harfbuzz.
```

Comment:
```suggestion
    /// shape horizontal LTR text without using Harfbuzz.
```

Reply:
Cool. That seems like a good clarification.